Makefile

```
@@ -0,0 +1,6 @@
+
+lint-cli: brew audit Formula/lacework-cli.rb --strict
+
+build-cli-from-source: brew install --build-from-source Formula/lacework-cli.rb 
+
+bump-cli-version: scripts/update-cli-version.sh
```

nitpick: we are "updating" the version, not bumping it.
```suggestion
update-cli-version: scripts/update-cli-version.sh
```